test/cliffordsynthesis/test_synthesis.cpp

```diff
@@ -70,9 +70,13 @@ class SynthesisTest : public ::testing::TestWithParam<TestConfiguration> {
       qc.import(ss, qc::Format::OpenQASM);
       std::cout << "Initial circuit:\n" << qc;
       targetTableau = Tableau(qc);
+      targetTableauWithDestabilizer =
+          Tableau(qc, 0, std::numeric_limits<std::size_t>::max(), true);
```

Comment:
Clang tidy complains about conversion from bool to unsigned, but it seemed to work when i tried it, so i'm not sure

Reply:
Ah, that's because
https://github.com/cda-tum/qmap/blob/c765e9c0065af1543ef527d08fdb31d0b15234dc/include/cliffordsynthesis/Tableau.hpp#L31-L33
defines all default arguments and takes precedence over the second constructor.
In that case, you can leave this as it is I guess. Or you could swap the order of the constructors, then it should also just work.

Reply:
Unfortunately it doesn't. Not exactly sure how the (my?) compiler determines precedence here. But it doesn't seem to agree with clang-tidy on this one.